### PR TITLE
Trivial spelling fix in _tkinter.c

### PR DESCRIPTION
**Pull request by AndrewGYork**
Change spelling of a #define in _tkinter.c from HAVE_LIBTOMMAMTH to HAVE_LIBTOMMATH, since this is used to keep track of tclTomMath.h, not tclTomMamth.h. No other file seems to refer to this variable.

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

## Recognized GitHub username

We couldn't find a [bugs.python.org](https://bugs.python.org/) (b.p.o) account corresponding to the following GitHub usernames:

@AndrewGYork

This might be simply due to a missing "GitHub Name" entry in one's b.p.o account settings. This is necessary for legal reasons before we can look at this contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.




You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

**Review by remilapeyre — APPROVED** [review]

**Comment by AndrewGYork** [comment]
Is this stalled because of something I'm doing wrong? Is there anything more I should do? This is my first contribution, perhaps I'm making a newbie mistake.

**Comment by remilapeyre** [comment]
Hi @AndrewGYork, thanks for contributing to Python !

There is a huge backlog of pull requests to be reviewed and only so much time core developers have to review them. Your changes look correct to me, the next step is to wait for a core developers to review the changes and he will either merge them or ask you for some modifications. They will get there but it may take some time, there is nothing you did wrong here.

I hope you are not too bothered by the time it takes to review PRs, if you are looking for things to do while waiting you can have a look at the other open issues at the bug tracker at bugs.python.org. Here's some issue marked as `easy`, `easy (c)` or `newcomer friendly`: https://bugs.python.org/issue?%40search_text=&ignore=file%3Acontent&title=&%40columns=title&id=&%40columns=id&stage=&creation=&creator=&activity=&%40columns=activity&%40sort=activity&actor=&nosy=&type=&components=&versions=&dependencies=&assignee=&keywords=18%2C6%2C-4%2C22%2C-4&priority=&status=1&%40columns=status&resolution=&nosy_count=&message_count=&%40group=&%40pagesize=50&%40startwith=0&%40sortdir=on&%40queryname=&%40old-queryname=&%40action=search

**Comment by AndrewGYork** [comment]
Thank you! I'm not bothered at all, and I'm glad to hear that I'm not making any obvious mistakes. Thank you for working on Python, I really appreciate it :-)

**Review by cjerdonek — APPROVED** [review]
LGTM. Thanks a lot for your contribution and for your patience, @AndrewGYork.

**Comment by cjerdonek** [comment]
Thanks again, Andrew.

**Comment by cjerdonek** [comment]
Oops, I had forgotten that `Misc/ACKS` and `Misc/NEWS.d` need to be updated even for trivial changes. I will go ahead and do that in a separate issue.

**Comment by cjerdonek** [comment]
The PR for the ACKS file update is here: #20105. Also, no NEWS entry was needed after all (only if there is an associated bpo issue).

**Comment by cjerdonek** [comment]
ACKS has been updated: https://github.com/python/cpython/commit/fe1176e882393b6d3e6a6cfa5ca23657f0b3b4a9

**Pull request merged**
